### PR TITLE
teacher tool: misc tidy stuff

### PR DESCRIPTION
**Pull request by eanders-ms**
Miscellaneous cleanup items I've been sitting on. I didn't want these included with other changes.

### CODE REVIEW

File: cli/cli.ts [hunk 1]
```diff
@@ -2047,7 +2047,6 @@ async function buildSemanticUIAsync(parsed?: commandParser.ParsedCommand) {
         generateReactCommonCss("skillmap"),
         generateReactCommonCss("authcode"),
         generateReactCommonCss("multiplayer"),
-        generateReactCommonCss("teachertool")
```
Comment (thsparks) [inline cli/cli.ts:2050]: I'm surprised. Did something change, or has this always been unnecessary?

File: teachertool/public/index.html [hunk 2]
```diff
@@ -7,17 +7,16 @@
     <meta name="theme-color" content="#000000" />
     <meta
       name="description"
-      content="MakeCode Teacher Tool. Automatically analyze and evaluate projects."
+      content="MakeCode Project Insights. Designed to help teachers evaluate student projects using a rubric."
```
Comment (thsparks) [inline teachertool/public/index.html:10]: FYI, don't need to block on this, but in our sync yesterday, there was some disagreement around the name Project Insights. We probably need to have a meeting with branding folks to hash this out.

File: cli/cli.ts (hunk 1 above)
Comment (eanders-ms) [reply cli/cli.ts:2050]: It's always been unnecessary, and was causing the react-common css to be loaded twice. Duplicates of every class.

**Review by thsparks — APPROVED** [review]

**Comment by eanders-ms** [comment]
I merged this rather quickly after opening it. @srietkerk please feel free to leave feedback if you had anything.

**Pull request merged**